### PR TITLE
Initial Receipt API

### PR DESCRIPTION
**Pull request by Kanokorn**
Provides Receipt API interfaces
`List all receipts` (Returns a list of receipt objects)
`Retrieve a receipt` (Returns an existing receipt object)

### CODE REVIEW

File: lib/omise/receipt.ex [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    currency:                nil
+  ]
+
+  @type t :: %__MODULE__{
+    object:                  String.t,
+    id:                      String.t,
+    number:                  String.t,
+    location:                String.t,
+    date:                    String.t,
+    customer_name:            String.t,
```
Comment (iporsut) [inline lib/omise/receipt.ex]: Indent?

Comment (Kanokorn) [reply lib/omise/receipt.ex]: 👌

File: lib/omise/receipt.ex [hunk 2]
```diff
@@ -0,0 +1,101 @@
+defmodule Omise.Receipt do
+  @moduledoc ~S"""
+  Provides Receipt API interfaces.
+
+  https:://www.omise.co/receipt-api
```
Comment (tteerawat) [inline lib/omise/receipt.ex]: Please insert `<>` so that this url can be click in the document website krub 👍

File: lib/omise/receipt.ex [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+  import Omise.HTTP
+
+  defstruct [
+    object:                  "receipt",
+    id:                      nil,
+    number:                  nil,
+    location:                nil,
+    date:                    nil,
+    customer_name:            nil,
```
Comment (tteerawat) [inline lib/omise/receipt.ex]: Indentation kruch.

File: lib/omise/receipt.ex (hunk 1 above)
Comment (Kanokorn) [reply lib/omise/receipt.ex]: @iporsut updated ka

**Review by tteerawat — APPROVED** [review]

**Review by iporsut — APPROVED** [review]

**Pull request merged**